### PR TITLE
Sync bundle manifests with deploy ones for v3.2.1

### PR DESCRIPTION
**Pull request by timdengyun**
Sync bundle manifests with latest manifests under directory deploy
for nsx-container-plugin-operator v3.2.1 version release in master branch

### CODE REVIEW

File: bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml
Comment (timdengyun) [inline bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:104]: It looks like here should be : docker.io/vmware/nsx-container-plugin-operator:v3.2.1? not sure how to generate it or edit version manually? 
The same question in code line #L22
name: nsx-container-plugin-operator.v3.2.0

Comment (salv-orlando) [reply bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:104]: This a bit of a chicken and egg problem, since the tagged image is generated upon release.
I'd say let's keep it untagged in master, and then we apply the tag on the release branch

Comment (timdengyun) [reply bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:104]: Hi Salvatore, Thanks for explaination. This patch will be merged into master, so I think that other code with v3.2.0 tag should be corrected  without tag as well, is it?  such as: 
code line #L11
containerImage: vmware/nsx-container-plugin-operator:3.2.0
code line #L22
name: nsx-container-plugin-operator.v3.2.0

I'm planning to cut release_v3.2.1 branch upon the PR merged, and then change image with tag v3.2.1.

Comment (salv-orlando) [reply bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:104]: That's correct. Your change does the right thing in my opinion

File: bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml
Comment (jwsui) [inline bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:11]: Maybe mistype here `vmware/nsx-container-plugin-operatorf`.

Comment (timdengyun) [reply bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml]: Hi Jianwei, thanks for correction. Fixed.

**Comment by jwsui** [comment]
LGTM, but it seems I'm  no authorized to approve this PR.

**Comment by timdengyun** [comment]
> LGTM, but it seems I'm no authorized to approve this PR.

Thanks jianwei,maybe you can raise a quest to add you as a maintainer for ncp operator repo so that you can get approval permission.

File: bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml
Comment (timdengyun) [reply bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml:104]: Hi Salvatore, if changes go well, could you help to approve this PR, Thanks a lot.

**Review by salv-orlando — APPROVED** [review]
Thanks Yun, changes LGTM!

**Pull request merged**
